app/client/src/git/sagas/fetchStatusSaga.ts

```
@@ -25,15 +27,22 @@ export default function* fetchStatusSaga(
         }),
       );
     }
-  } catch (error) {
-    yield put(
-      gitArtifactActions.fetchStatusError({
-        ...basePayload,
-        error: error as string,
-      }),
-    );
+  } catch (e) {
+    if (response && response.responseMeta.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)